src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -365,6 +365,9 @@ public void CSharp_VerifyIDEDiagnosticSeveritiesAreConfigurable()
 # IDE0073
 dotnet_diagnostic.IDE0073.severity = %value%
 
+# IDE0074
+dotnet_style_prefer_compound_assignment = true:suggestion
```

Comment:
Seems like the same option controls even `IDE0054`? Is the only reason this has a different diagnostic ID is because it is implemented in a different analyzer? If so, would it make sense to merge this into IDE0054 analyzer with a distinct analyzer action that reports the same diagnostic ID?